src/js/pages/services/DeploymentsTab.js

```diff
@@ -228,7 +228,7 @@ class DeploymentsTab extends mixin(StoreMixin) {
         linkText = linkText + ' & Remove';
       }
       return (
-        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover"
+        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover pull-right"
```

Comment:
Agreed, `pull-right` does exist in Canvas, but it makes use of `float: right;` which I am not a fan of. Though you could argue this is personal preference...